src/worker/tasks/send-transaction-worker.ts

```diff
@@ -143,7 +143,14 @@ const _sendUserOp = async (
 
   assert(accountAddress, "Invalid userOp parameters: accountAddress");
   const toAddress = to ?? target;
-  assert(toAddress, "Invalid transaction parameters: to");
+
+  if (queuedTransaction.batchOperations) {
+    queuedTransaction.batchOperations.map((op) => {
+      assert(op.to, "Invalid transaction parameters: to");
+    });
+  } else {
+    assert(toAddress, "Invalid transaction parameters: to");
+  }
```

Comment:
For my understanding:
- in a single userOp, `toAddress` is set.
- in a batch userOp, `toAddress` is not set but it must be set on every batch operation.

What about contract deploys? Is there a toAddress on a userOp to deploy a contract?

Reply:
for contract deployment, it should be `0x0000000000000000000000000000000000000000`

Reply:
I thought contract deployment `to = null`? Something to test